packages/maskbook/src/plugins/BarnBridge/Import/BarnBridgeFrontEnd/PortfolioBalance.tsx

```diff
@@ -0,0 +1,100 @@
+import { makeStyles } from '@masknet/theme'
+
+// import { withStyles } from "@material-ui/core/styles";
```

Comment:
I ran into this being blocked by lint rules -> is there any alternative masknet prefers. makeStyles isn't really what I want here....